core/chains/evm/types/block_hash.go

```diff
@@ -0,0 +1,47 @@
+package types
+
+import (
+	"bytes"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	commontypes "github.com/smartcontractkit/chainlink/v2/common/types"
+)
+
+type BlockHash struct {
+	commontypes.Hashable
```

Comment:
Is this meant to be here?

Reply:
This seems to be a recurring pattern, but it doesn't make sense to me. Why are we embedding a variable of the type we are adapting the native type to?